src/ensembl/src/content/app/browser/track-panel/trackPanelActions.ts

```diff
@@ -101,17 +188,63 @@ export const changeHighlightedTrackId: ActionCreator<
     updateTrackPanelForGenome({
       activeGenomeId,
       data: {
-        ...getActiveTrackPanel(getState()),
+        ...getActiveTrackPanel(state),
         highlightedTrackId
       }
     })
   );
 };
 
+// FIXME: Wait until manual bookmarks functionality is added
+// export const updateBookmarksAndSave: ActionCreator<
+//   ThunkAction<void, any, null, Action<string>>
+// > = () => (dispatch, getState: () => RootState) => {
+//   const state = getState();
+//   const activeGenomeId = getBrowserActiveGenomeId(state);
+//   const activeEnsObject = getBrowserActiveEnsObject(state);
+//   if (!activeGenomeId || !activeEnsObject) {
+//     return;
+//   }
+//   const trackStates = getBrowserTrackStates(state)[activeGenomeId];
+
+//   const activeGenomeBookmarks = [...getActiveGenomeBookmarks(state)];
+
+//   const existingIndex = activeGenomeBookmarks.findIndex(
+//     (bookmark) => bookmark.object_id === activeEnsObject.object_id
+//   );
+//   if (existingIndex === -1) {
+//     // IF it is not present, add it to the end
+//     activeGenomeBookmarks.push(buildBookmark(activeEnsObject, trackStates));
+//   } else if (existingIndex !== -1) {
+//     // If it is already present, bump it to the end
+//     activeGenomeBookmarks.push({
+//       ...activeGenomeBookmarks[existingIndex],
+//       trackStates
+//     });
+//     activeGenomeBookmarks.splice(existingIndex, 1);
+//   }
+
+//   trackPanelStorageService.updateActiveGenomeBookmarks({
+//     [activeGenomeId]: activeGenomeBookmarks
+//   });
+
+//   dispatch(
+//     updateTrackPanelForGenome({
+//       activeGenomeId,
+//       data: {
+//         ...getActiveTrackPanel(getState()),
+//         bookmarks: activeGenomeBookmarks
+//       }
+//     })
+//   );
+// };
```

Comment:
Perhaps remove all this commented code? It probably is not correct anyway (we probably were not supposed to cap bookmarks at 20)

Reply:
That's true. Remove it.

Reply:
That's true. Removed it.